www_src/pages/element/element.jsx

```diff
@@ -20,7 +20,10 @@ var testIds = {
 };
 
 render(React.createClass({
-  mixins: [router],
+  mixins: [
+    router,
```

Comment:
Nitpick: Moving the `require` into the `mixins` array as per the new `snackbar-mixin` is a good idea. Would be nice to have consistency.